create_jenkins_job.py

```diff
@@ -150,10 +157,17 @@ def main(argv=None):
         if os_name == 'win':
             job_name = job_name[0:-2]
         job_data['cmake_build_type'] = 'Debug'
+	# For the aarch64 job, don't email on test failures (because there are
```

Comment:
Please do not use tabs in Python code: fixed in c1180319d10cf166bfad16400b51f07bf7f6d8e4